js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -570,7 +569,7 @@ export const LAUNCH_ASSET_EXECUTION_ASSET_NODE_FRAGMENT = gql`
   ${ASSET_NODE_CONFIG_FRAGMENT}
 `;
 
-export const LAUNCH_ASSET_LOADER_QUERY = gql`
+export const LAUNCH_ASSET_LOADER_QUERY = graphql(`
```

Comment:
Hmm do we need the next line to be just `LaunchAssetLoader` to get a TS type named `LaunchAssetLoaderQuery`? It's weird because they seem to say `query AQuery {` in all their examples, but I can't imagine the `QueryQuery` type is the intended naming?